### ISSUE TITLE
Quick access: editor picker: Editor dirty state is not announced to screen reader

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#93936]
#93637

Version: 1.44.0-insider (user setup)
Commit: d7d1147d6b764a48b789000c03088ee9afabcd8a
Date: 2020-03-31T08:23:56.446Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18362

---

This state:

![image](https://user-images.githubusercontent.com/2193314/78030905-abae7280-7317-11ea-9a43-c52680fef2db.png)

Announced: "Picker quick access dot ts editor group picker 3 of 5"

This state:

![image](https://user-images.githubusercontent.com/2193314/78031048-d4cf0300-7317-11ea-88e4-691692be43ac.png)

Announced "Toolbar, close editor action"

**Comment by Tyriar** [comment]
When focusing the button on the recent workspace feature it announces dirty, however it should do this when you're navigating the list, not when just navigating between items. Otherwise the only way to tell if it has backups/changes is to tab into close the editor.

![image](https://user-images.githubusercontent.com/2193314/78031861-fb416e00-7318-11ea-9c8f-2ea656d15842.png)

**Comment by bpasero** [comment]
Needs verification in:
* editor picker (`edt`)
* files picker (e.g. having a dirty editor in history)
* window switcher (Window > Switch Window) when having dirty files in that window
* recently opened (when having dirty files in workspace)
